### DISCUSSION TITLE
share ur cool tips

### DISCUSSION THREAD

**Discussion by Amin456789** [LostRuins/koboldcpp#908] [General]
hi, here we share things we find about this great app

1. make sure if u r using cpu let the app choose ur cpu cores, for example if it says 2 let it be 2, dont make it 4. my cpu for laptop is 4 cores and when i put it on 4 it made the image generation worse, imake 1 step image 512x512 with sdxl turbo in 30 seconds in 2 cores but when i put in 4 it was in 48 second
3. quantize the safetensors of image generation to save space = i quantized sdxl turbo to q5.0 and it became only 2.8gb and results are the same as q8 and great quality = u can use stable-diffusion.cpp repo to quantize them
4. im using h2o danube q4 1.8b model which is one of the best models out there for text generation, very fast and ram friendly and great in logic and general information of world  and it beats some 7b and 3b models too
5. make sure to have another web browser for not loosing data = i use portable version of brave for this and sd ui

**Comment by nielsijzerman** [comment]
1. System needs cpu - interupts messed up your render times I bet
2. What happened to your speed?
3. I like big B's and can't deny
4. And one dedicated for porn, pc as well as browser.

My tips: Don't value sleep too much. Don't watch tv. Don't work out. Or work, for that matter, you need the time benchmarking, troubleshooting, crawling the web, browsing AliX,/Amazon/CraigsList/local hardware shops inventory, cursing the U in USB for never really being Universal; the nicer things in life. Don't value free space in your (bed)room/appartement/house/attic/shed,and above all, don't value money in the bank over equipment. 

Always get more storage space when you can; ssd's preferably, but (free) hdd's are great in RAID. When you put 6 together in RAID 0, they're like a poor mans ssd. Laptop disks are silent and run cool. Small ssd's put together make a bigger, faster one. Use software RAID, unless you have patience restoring the mess after power failure, and sometimes even afloppy disk drive for the drivers.

You can't have enough sata/usb/pcie/hdmi/dp/dvi ports, cables, connectors, adapters etc.

2nd hand flatscreens are dirt cheap, I bought some from the thrift store just for the stand. €5 for an ips panel - I mean, come on.
